mender-convert

```diff
@@ -266,9 +263,21 @@ do_raw_disk_image_create_partitions() {
 
   mkdir -p $output_dir && cd $output_dir
 
-  # In case of missing .sdimg name use the default format.
-  [ -z $mender_disk_image ] && mender_disk_image=$output_dir/mender_${device_type}.sdimg \
-                            || mender_disk_image=$output_dir/$mender_disk_image
+  # Make sure the user's given Mender image name has a correct extension.
+  # If Mender image name is not provided, then use following syntax:
+  # mender-<device_name>-<artifact_name>.sdimg
+  if [ -n "${mender_disk_image}" ]; then
```

Comment:
I am not really sure what this options gives us, beside adding unnecessary complexity. I would simply drop this option. 

The user can and must set the `artifact_name` and that should be enough as customization and separation from previous builds as we require the `artifact_name` to be unique.

Reply:
I do not follow, is this variable used for different things based on which command you run?

Reply:
When _from-raw-disk-image_ is called, then only the desired name for Mender image must be provided. But when you run _mender-disk-image-to-artifact_ or expert_ commands_ like _install-mender-to-mender-disk-image_ & _install-bootloader-to-mender-disk-image_ then _mender-disk-name_ passed in the CLI must be the path. _mender_disk_image_ variable always holds the path to the Mender image.